interfaces/PA-App/src/app/personal-components/preprinted-qrcode/preprinted-qrcode.component.html

```diff
@@ -1,4 +1,4 @@
-<section *ngIf="isCanceled === false">
+<section *ngIf="!isCanceled">
```

Comment:
I think the `=== false` was put there on purpose, because it doesn't match when it it is `undefined`...

Reply:
@elwinschmitz sure.. but it wasn't working like this, because the component wasn't showing while it should.

Reply:
@elwinschmitz just found out that the same change is causing also the store-credential component to not show.

that being said:
- I do admit that I haven't tested the reverse yet with this change (program without validation)
- and I am interested to hear more about it. Because if you think it's needed for some purpose, while I think the reverse is needed for some purpose, that should be straightened out.